deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,672 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.11.2
+  namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.ppc64le: supported
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: quay.io/ibmcsiblock/ibm-block-csi-operator:1.11.2
+    createdAt: "2022-03-31T11:00:00Z"
```

Comment:
maybe we should update this time stamp

Reply:
Command to find all "createdAt" under 1.11.2 directories:
```
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> for diri in $(find . -name 1.11.2 -type d); do grep -nR createdAt $diri; done
./deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:12:    createdAt: "2024-04-15T11:00:00Z"
./deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:15:    createdAt: "2024-04-15T11:00:00Z"
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator>
```